src/tests/JIT/Regression/JitBlue/Runtime_106338/Runtime_106338.cs

```diff
@@ -8,15 +8,22 @@
 // Debug: Outputs 1600094603
 // Release: Outputs 1600094604
 using System;
+using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics.X86;
 using Xunit;
 
 public class Runtime_106338
 {
     [Fact]
     public static void TestEntryPoint()
     {
-        ulong vr10 = 16105307123914158031UL;
-        float vr11 = 4294967295U | vr10;
-        Assert.Equal(1600094603U, BitConverter.SingleToUInt32Bits(vr11));
+        bool runTest = (RuntimeInformation.ProcessArchitecture == Architecture.Arm64) || Avx512F.IsSupported;
+
+        if (runTest)
+        {
+            ulong vr10 = 16105307123914158031UL;
+            float vr11 = 4294967295U | vr10;
+            Assert.Equal(1600094603U, BitConverter.SingleToUInt32Bits(vr11));
+        }
```

Comment:
Should the test rather assert that its producing `1600094604u` if `(RuntimeInformation.ProcessArchitecture == Architecture.Arm64) || Avx512F.IsSupported` is `false`?

That way we can detect any changes for other platforms or scenarios?

Reply:
I just remembered that we don't build this test if the target arch isn't arm64/x64 (or if the runtime isn't CoreCLR). @tannergooding are you ok with only testing those platforms for now?

Reply:
I think that’s fine, but ideally we’d ensure this runs everywhere long term